roles/tempest/defaults/main.yml

```diff
@@ -28,7 +28,7 @@ cifmw_tempest_container: openstack-tempest
 cifmw_tempest_image: "{{ cifmw_tempest_registry }}/{{ cifmw_tempest_namespace }}/{{ cifmw_tempest_container }}"
 cifmw_tempest_image_tag: current-podified
 cifmw_tempest_dry_run: false
-cifmw_tempest_remove_container: false
+cifmw_tempest_remove_container: true
```

Comment:
why this change?

Reply:
@slawqo please go to this conversation:
https://github.com/openstack-k8s-operators/ci-framework/pull/1013/files#r1448599697

Cc @cjeanner